extensions/cornerstone-dicom-rt/src/viewports/_getStatusComponent.tsx

```diff
@@ -34,13 +34,10 @@ export default function _getStatusComponent({ isHydrated, onStatusClick }) {
           <span className="ml-1">RTSTRUCT</span>
         </div>
         {!isHydrated && (
-          <div
-            className="bg-primary-main hover:bg-primary-light ml-1 cursor-pointer rounded px-1.5 hover:text-black"
-            // Using onMouseUp here because onClick is not working when the viewport is not active and is styled with pointer-events:none
-            onMouseUp={onStatusClick}
-          >
-            {loadStr}
-          </div>
+          <ViewportActionButton
```

Comment:
Isn't this definition supposed to come from the toolbar definition?  That way the toolbar can have additional viewport action buttons - one for load, and a second for add

Reply:
Please disregard - I thought it was SR, not RT